backends/backends.c

```diff
@@ -301,6 +301,10 @@ void *backends_main(void *ptr) {
 
     global_backend_options = backend_parse_data_source(source, global_backend_options);
 
+#ifdef ENABLE_HTTPS
+    struct netdata_ssl opentsdb_ssl = { NULL , NETDATA_SSL_START};
```

Comment:
What is the logic of placing the struct declaration here? It looks like a randomly chosen place for me.

Reply:
I followed a conclusion that I arrived after to study the function backends_main(), the majority of the variables are declared before the main loop, so I decided to put this variable in this position. It could be nearest the main loop without any affect  to the code, so I can say that it is "almost" random.

Reply:
I'd move it [here](https://github.com/netdata/netdata/blob/f2d5c1b7ee36574e3c8fd4cea3d86ab325e28792/backends/backends.c#L458).

Reply:
All right, it will be in this line in the next commit.